<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsChangeWidget</class>
 <widget class="QWidget" name="SettingsChangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="OPNWidget" name="fmWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="NoiseWidget" name="noiseWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="SSGWidget" name="ssgWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="MelodyWidget" name="melodyWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_6">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="RhythmWidget" name="rhythmWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_7">
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="PCMWidget" name="romWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="SettingsChangeFooterWidget" name="footerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>84</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>84</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PCMWidget</class>
   <extends>QWidget</extends>
   <header>pcm/pcmwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OPNWidget</class>
   <extends>QWidget</extends>
   <header>opn/opnwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NoiseWidget</class>
   <extends>QWidget</extends>
   <header>psg/noisewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsChangeFooterWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/pianorollwidget/settingschangefooterwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SSGWidget</class>
   <extends>QWidget</extends>
   <header>ssg/ssgwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MelodyWidget</class>
   <extends>QWidget</extends>
   <header>opl/melodywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RhythmWidget</class>
   <extends>QWidget</extends>
   <header>opl/rhythmwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
